src/platform/user/profile/vap-svc/components/ContactInfoForm.jsx

```diff
@@ -17,6 +17,7 @@ const ContactInfoForm = props => (
     }}
     onSubmit={e => props.onSubmit(e)}
     data={props.formData}
+    stopPropagation
```

Comment:
I have mild concerns about this being hardcoded in. Can this bool be passed into `ContactInfoForm` as a prop?

Reply:
Sure, I can do that.

Reply:
implemented!

Reply:
Now I've removed all `stopPropagation` props... sorry!